zoltan/apfZoltanCallbacks.cc

```diff
@@ -29,9 +31,33 @@ static int setZoltanLbMethod(struct Zoltan_Struct* ztn, ZoltanMesh* zb)
     case HYPERGRAPH:
       lbMethod = "HYPERGRAPH"; break;
     case PARMETIS: //fall into GRAPH settings
+      lbMethod = "GRAPH";
+#ifdef PUMI_HAS_PARMETIS
+      Zoltan_Set_Param(ztn, "GRAPH_PACKAGE", "PARMETIS");
+#else
+      lion_oprint(1, "WARNING: ParMETIS ZoltanMethod requested but ParMETIS"
+        " was not enabled at build time.\n");
```

Comment:
I think we should make this an error (i.e., stop execution) or, indicate to the user that PHG will be used instead (IIUC): 
https://sandialabs.github.io/Zoltan/ug_html/ug_alg_graph.html

Also, `lion_oprint` will write on all ranks; we should probably only print from rank 0 here.